### PR TITLE
Fix memory leak

### PR DESCRIPTION
**Pull request by Gordon-F**
fix some memory leak

### CODE REVIEW

**Comment by punto-** [comment]
All the process' memory is released on exit, so it's not really a leak (at
least not for long :p )

File: platform/windows/godot_win.cpp [hunk 1]
```diff
@@ -137,8 +137,13 @@ int widechar_main(int argc, wchar_t** argv) {
 
 	Error err = Main::setup(argv_utf8[0], argc - 1, &argv_utf8[1]);
 
-	if (err!=OK)
+	if (err!=OK) {
+		for (int i=0; i<argc; ++i) {
+			delete[] argv_utf8[i];
```
Comment (bojidar-bg) [inline platform/windows/godot_win.cpp]: Isn't `delete[]` a bit too C++11 for us?

Comment (vnen) [inline platform/windows/godot_win.cpp]: `delete[]` is present in C++98 standard. I don't see the point of freeing the memory just before exiting though.

**Comment by Gordon-F** [comment]
@vnen agree. Close PR.

**Pull request closed without merging**